ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -1707,6 +1708,7 @@ static int prdcr_unsubscribe_regex_handler(ldmsd_req_ctxt_t reqc)
 	ldmsd_send_req_response(reqc, reqc->line_buf);
 	if (prdcr_regex)
```

Comment:
This check is unnecessary. free() correctly handles NULL.

Reply:
the patch is to fix leaks, not clean up  pre-existing style issues.

Reply:
This is not a style issue; the check is unnecessary and the change is right next to yours. As we find these little nits whether or not this change introduced them, we should fix them. Please fix.

Reply:
As there is no reasonable limit to "as we find" and "right next to" in unrelated nit fixing, I simply stopped at the file scope.